README.rst

```diff
@@ -18,52 +18,120 @@ python-zyte-api
    :target: https://codecov.io/gh/zytedata/zyte-api
    :alt: Coverage report
 
-Python client libraries for `Zyte API`_.
+.. description-start
 
-Command-line utility and asyncio-based library are provided by this package.
+Command-line client and Python client library for `Zyte API`_.
+
+.. _Zyte API: https://docs.zyte.com/zyte-api/get-started.html
+
+.. description-end
 
 Installation
 ============
 
-::
+.. install-start
 
-    pip install zyte-api
-
-``zyte-api`` requires Python 3.8+.
+.. code-block:: shell
 
-API key
-=======
+    pip install zyte-api
 
-Make sure you have an API key for the `Zyte API`_ service.
-You can set ``ZYTE_API_KEY`` environment
-variable with the key to avoid passing it around explicitly.
+.. note:: Python 3.8+ is required.
 
+.. install-end
 
 Basic usage
 ===========
 
-Use the ``zyte-api`` command to send Zyte API requests from the command line:
+.. basic-start
+
+Set your API key
+----------------
+
+.. key-get-start
+
+After you `sign up for a Zyte API account
+<https://app.zyte.com/account/signup/zyteapi>`_, copy `your API key
+<https://app.zyte.com/o/zyte-api/api-access>`_.
+
+.. key-get-end
+
+Then configure your Zyte API key as an environment variable in your current
```

Comment:
I think to simplify the README we can show an example where the key is passed as an argument.

Reply:
Though no strong opinion on that; it's also fine as-is.